File: frontend/src/types/protocol.entry.ts
export default interface IProtocolEntryData {
    id?: number | null,
    testProtocolId: number,
    source: string,
    level: string,
    message: string,
    data: string,
    runtime: number,
    createdAt: Date,
    updatedAt: Date,
    deletedAt: Date,
}

File: storage/models/protocol_entry.go
package models

import (
	"time"
)

type ProtocolEntry struct {
	Model
	TestProtocolID uint      `json:"testProtocolId"`
	Timestamp      time.Time `json:"timestamp"`
	Source         string    `json:"source"`
	Level          string    `json:"level"`
	Message        string    `json:"message"`
	Data           string    `json:"data"`
	Runtime        float64   `json:"runtime"`
}
